notebooks/catvton/catvton.ipynb

```diff
@@ -0,0 +1,261 @@
+    " 2. Parameter-efficient training. We identified the try-on relevant modules through experiments and achieved high-quality try-on effects by training only 49.57M parameters (∼5.51% of the backbone network’s parameters). \n",
+    " 3. Simplified inference. CatVTON eliminates all unnecessary conditions and preprocessing steps, including pose estimation, human parsing, and text input, requiring only garment reference, target person image, and mask for the virtual try-on process. Extensive experiments demonstrate that CatVTON achieves superior qualitative and quantitative results with fewer prerequisites and trainable parameters than baseline methods. Furthermore, CatVTON shows good generalization in in-the-wild scenarios despite using open-source datasets with only 73K samples.\n",
+    "\n",
+    "\n",
+    "Teaser image from [CatVTON GitHub](https://github.com/Zheng-Chong/CatVTON)\n",
+    "![teaser](https://github.com/Zheng-Chong/CatVTON/blob/edited/resource/img/teaser.jpg?raw=true)\n",
+    "\n",
+    "In this tutorial we consider how to convert, optimize and run this model using OpenVINO.\n",
+    "\n",
+    "\n",
+    "#### Table of contents:\n",
+    "\n",
+    "- [Prerequisites](#Prerequisites)\n",
+    "- [Convert the model to OpenVINO IR](#Convert-the-model-to-OpenVINO-IR)\n",
+    "- [Compiling models](#Compiling-models)\n",
+    "- [Interactive demo](#Interactive-demo)\n",
+    "\n",
+    "\n",
+    "### Installation Instructions\n",
+    "\n",
+    "This is a self-contained example that relies solely on its own code.\n",
+    "\n",
+    "We recommend  running the notebook in a virtual environment. You only need a Jupyter server to start.\n",
+    "For details, please refer to [Installation Guide](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/README.md#-installation-guide).\n",
+    "\n",
+    "<img referrerpolicy=\"no-referrer-when-downgrade\" src=\"https://static.scarf.sh/a.png?x-pxid=5b5a4db0-7875-4bfb-bdbd-01698b5b1a77&file=notebooks/catvton/catvton.ipynb\" />\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b91d041-f139-47c7-bca8-805249648b6f",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "[back to top ⬆️](#Table-of-contents:)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a1f0cac2-0661-4974-a5c6-34d0bed7c92c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -q \"openvino>=2024.4\"\n",
+    "%pip install -q \"torch>=2.1\" \"diffusers>=0.29.1\" torchvision opencv_python \"numpy<2\" --extra-index-url https://download.pytorch.org/whl/cpu\n",
+    "%pip install -q fvcore \"pillow\" \"tqdm\" \"gradio>=4.36\" \"omegaconf==2.4.0.dev3\" av pycocotools cloudpickle scipy accelerate \"transformers>=4.27.3\""
```

Comment:
you said that notebook demostrates how to optimize, but I do not see nncf in requirements (under optimization we usually means quantization). Is there is some plans for that or optimization should be removed from description until this is not added?

scipy package is heavily depends from numpy, could you please check that it will not breaks if you do  not specify lower bound for scipy and have it for numpy?